src/main/java/com/android/volley/Cache.java

```diff
@@ -87,12 +87,12 @@
         public Map<String, String> responseHeaders = Collections.emptyMap();
 
         /** True if the entry is expired. */
-        public boolean isExpired() {
+        boolean isExpired() {
```

Comment:
I missed that these are methods in a class, not an interface. This was a functional change that should be reverted. (I'll take care of this)